modules/migration-DNS-considerations.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+
+[id="migration-DNS-considerations_{context}"]
+== DNS considerations
+
+The DNS domain of the OpenShift Container Platform 4 target cluster is different from the domain of the OpenShift Container Platform 3 source cluster. By default, applications get new FQDNs after migration.
```

Comment:
Use attribute for product name

Reply:
`applications get new FQDNs after migration.` You can be more specific. They get the FQDN of the OCP 4 cluster.